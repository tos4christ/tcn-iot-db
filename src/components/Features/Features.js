import React from "react";
import Footer from "../Footer/Footer";
import { Element } from "react-scroll";
import feature from "../../assets/img/feature.png";

const Features = () => {
  return (
    <Element id="features" name="features">
      <section className="feature-area ptb-90" id="feature">
        <div className="container">
          <div className="sec-title">
            <h2 className="text-light">Features</h2>
          </div>
          <div className="row flexbox-center">
            <div className="col-lg-4">
              <div className="single-feature-box text-lg-right text-center">
                <ul>
                  <li>
                    <div className="feature-box-info">
                      <h4>Easy to Use</h4>
                    </div>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-brush"></i>
                    </div>
                  </li>
                  <li>
                    <div className="feature-box-info">
                      <h4>Responsive Design</h4>
                    </div>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-computer"></i>
                    </div>
                  </li>
                  <li>
                    <div className="feature-box-info">
                      <h4>Well Documented</h4>
                    </div>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-law-document"></i>
                    </div>
                  </li>
                  <li>
                    <div className="feature-box-info">
                      <h4>Light Weight</h4>
                    </div>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-heart-beat"></i>
                    </div>
                  </li>
                </ul>
              </div>
            </div>
            <div className="col-lg-4">
              <div className="single-feature-box text-center jump">
                <img src={feature} alt="feature" />
              </div>
            </div>
            <div className="col-lg-4">
              <div className="single-feature-box text-lg-left text-center">
                <ul>
                  <li>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-eye"></i>
                    </div>
                    <div className="feature-box-info">
                      <h4>Night Mode</h4>
                    </div>
                  </li>
                  <li>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-sun-alt"></i>
                    </div>
                    <div className="feature-box-info">
                      <h4>Works Offline</h4>
                    </div>
                  </li>
                  <li>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-code-alt"></i>
                    </div>
                    <div className="feature-box-info">
                      <h4>Real Time Updates</h4>
                    </div>
                  </li>
                  <li>
                    <div className="feature-box-icon">
                      <i className="icofont icofont-headphone-alt"></i>
                    </div>
                    <div className="feature-box-info">
                      <h4>Open to Feedbacks</h4>
                    </div>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
        <Footer />
      </section>
    </Element>
  );
};

export default Features;
