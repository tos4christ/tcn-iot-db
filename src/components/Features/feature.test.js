import React from "react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";
import Features from "./Features";

it("renders features component", () => {
  let tree = renderer.create(<Features/>).toJSON()
  expect(tree).toMatchSnapshot()
});
