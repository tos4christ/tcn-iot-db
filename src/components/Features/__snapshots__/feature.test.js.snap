// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders features component 1`] = `
<div
  id="features"
  name="features"
>
  <section
    className="feature-area ptb-90"
    id="feature"
  >
    <div
      className="container"
    >
      <div
        className="sec-title"
      >
        <h2
          className="text-light"
        >
          Features
        </h2>
      </div>
      <div
        className="row flexbox-center"
      >
        <div
          className="col-lg-4"
        >
          <div
            className="single-feature-box text-lg-right text-center"
          >
            <ul>
              <li>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Easy to Use
                  </h4>
                </div>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-brush"
                  />
                </div>
              </li>
              <li>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Responsive Design
                  </h4>
                </div>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-computer"
                  />
                </div>
              </li>
              <li>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Well Documented
                  </h4>
                </div>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-law-document"
                  />
                </div>
              </li>
              <li>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Light Weight
                  </h4>
                </div>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-heart-beat"
                  />
                </div>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-lg-4"
        >
          <div
            className="single-feature-box text-center jump"
          >
            <img
              alt="feature"
              src="feature.png"
            />
          </div>
        </div>
        <div
          className="col-lg-4"
        >
          <div
            className="single-feature-box text-lg-left text-center"
          >
            <ul>
              <li>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-eye"
                  />
                </div>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Night Mode
                  </h4>
                </div>
              </li>
              <li>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-sun-alt"
                  />
                </div>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Works Offline
                  </h4>
                </div>
              </li>
              <li>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-code-alt"
                  />
                </div>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Real Time Updates
                  </h4>
                </div>
              </li>
              <li>
                <div
                  className="feature-box-icon"
                >
                  <i
                    className="icofont icofont-headphone-alt"
                  />
                </div>
                <div
                  className="feature-box-info"
                >
                  <h4>
                    Open to Feedbacks
                  </h4>
                </div>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      className="copyright-area"
      style={
        Object {
          "marginBottom": "-50px",
          "marginTop": "90px",
        }
      }
    >
      <p>
        Copyright Â©
        2020
         | This web app is made with
         
        <i
          aria-hidden="true"
          className="fa fa-heart-o"
        />
         by Team-003
      </p>
    </div>
  </section>
</div>
`;
